clients/python/client/osparc/_models.py

```diff
@@ -0,0 +1,20 @@
+from uuid import UUID
+
+from pydantic import Field, field_validator
+from pydantic_settings import BaseSettings
+
+
+class ParentProjectInfo(BaseSettings):
+    """Information a project cann pass onto its "children" (i.e. projects
+    'spawned' through the api-server)"""
+
+    x_simcore_parent_project_uuid: str | None = Field(
```

Comment:
Q: why not define the type directly as UUID?

Reply:
Ultimately I need these fields as strings when I pass them onto the webserver. So instead of having to convert them manually into strings there I prefer to do it here, but validate that those strings are valid uuids.